src/ActionMonitor/Monitors/PreviewMonitor.php

```diff
@@ -29,8 +29,8 @@ static function get_gatsby_preview_webhook() {
 			return false;
 		}
 
-		if ( substr( $preview_webhook, - 1 ) !== '/' ) {
-			$preview_webhook = "$preview_webhook/";
+		if ( substr( $preview_webhook, -1 ) !== '/' ) {
+			$preview_webhook .= '/';
```

Comment:
Maybe we can call this `trailingslashit`
https://developer.wordpress.org/reference/functions/trailingslashit/
What do you think?

Reply:
😆